samples/MyLibrary/OutsideNamespaceClass.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Localization;
+
+namespace OtherNamespace
+{
+    public class OutsideNamespaceClass
```

Comment:
`OtherNamespaceClass`. `OutsideNamespace` sounds like it's a type in the global namespace